testing/python/metafunc.py

```diff
@@ -784,7 +784,6 @@ def pytest_generate_tests(metafunc):
 
             @pytest.fixture
             def metafunc(request):
-                assert request._pyfuncitem._genid == "0"
```

Comment:
now that one is interesting

Reply:
it is interesting!  here's the last usage of that one: 426907eafbba7738cf33374a33c3c7aed67dadce

Reply:
yikes, thanks